dist/mac/Info.plist

```diff
@@ -90,5 +98,10 @@
 			</dict>
 		</dict>
 	</array>
+	<key>NSAppTransportSecurity</key>
```

Comment:
I still think it is better if we can leave this out, we haven't receive any complains about it...
You can try arch linux ISO magnet link: https://www.archlinux.org/download/